bin/cli-flags.js

```diff
@@ -1,427 +1,854 @@
+        multiple: false,
+        path: 'allowedHosts',
+        type: 'enum',
+        values: ['auto', 'all'],
+      },
+    ],
+    description:
+      "Defines routes which are enabled by default, on by default and allows localhost/value from the 'host' option/value from the 'client.webSocketURL' option. https://webpack.js.org/configuration/dev-server/#devserverallowedhosts",
+    multiple: true,
+    simpleType: 'string',
+  },
+  'allowed-hosts-reset': {
+    configs: [
+      {
+        type: 'reset',
+        multiple: false,
+        description:
+          "Clear all items provided in configuration. Defines routes which are enabled by default, on by default and allows localhost/value from the 'host' option/value from the 'client.webSocketURL' option. https://webpack.js.org/configuration/dev-server/#devserverallowedhosts",
+        path: 'allowedHosts',
+      },
+    ],
+    description:
+      "Clear all items provided in configuration. Defines routes which are enabled by default, on by default and allows localhost/value from the 'host' option/value from the 'client.webSocketURL' option. https://webpack.js.org/configuration/dev-server/#devserverallowedhosts",
+    simpleType: 'boolean',
+    multiple: false,
+  },
+  bonjour: {
+    configs: [
+      {
+        type: 'boolean',
+        multiple: false,
+        description:
+          'Broadcasts the server via ZeroConf networking on start. https://webpack.js.org/configuration/dev-server/#devserverbonjour',
+        path: 'bonjour',
+      },
+    ],
+    description:
+      'Broadcasts the server via ZeroConf networking on start. https://webpack.js.org/configuration/dev-server/#devserverbonjour',
+    negatedDescription:
+      'Do not broadcast the server via ZeroConf networking on start.',
+    simpleType: 'boolean',
+    multiple: false,
+  },
+  'client-transport': {
+    configs: [
+      {
+        type: 'enum',
+        values: ['sockjs', 'ws'],
+        multiple: false,
+        description:
+          'Allows to set custom transport to communicate with server.',
+        path: 'client.transport',
+      },
+      {
+        type: 'string',
+        multiple: false,
+        description:
+          'Allows to set custom transport to communicate with server.',
+        path: 'client.transport',
+      },
+    ],
+    description: 'Allows to set custom transport to communicate with server.',
+    simpleType: 'string',
+    multiple: false,
+  },
+  'client-logging': {
+    configs: [
+      {
+        type: 'enum',
+        values: ['none', 'error', 'warn', 'info', 'log', 'verbose'],
+        multiple: false,
+        description:
+          'Specifies client properties. https://webpack.js.org/configuration/dev-server/#devserverclient',
+        path: 'client.logging',
+      },
+    ],
+    description:
+      'Specifies client properties. https://webpack.js.org/configuration/dev-server/#devserverclient',
+    simpleType: 'string',
+    multiple: false,
```

Comment:
So we don't need `processor` anymore? for older cli versions?

Reply:
Yes, we don't need this, we will work like webpack